### PR TITLE
Added ability to sync any property with any property.

### PR DESCRIPTION
**Pull request by PascalPiche**
…r outerHeight and height.

### CODE REVIEW

File: src/modules/auto-sync-property.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /**
  * @author Deux Huit Huit
- * 
+ *
```
Comment (nitriques) [inline src/modules/auto-sync-property.js]: Please add a description

File: src/modules/auto-sync-property.js [hunk 2]
```diff
@@ -20,6 +21,7 @@
 		if (property) {
 			var sourceSelector = t.attr(SOURCE_ATTR);
 			var commonAncestorSelector = t.attr(COMMON_ANCESTOR_ATTR);
+			var withProperty = t.attr(WITH_PROPERTY_ATTR) || '';
```
Comment (nitriques) [inline src/modules/auto-sync-property.js]: We can drop the ` || ''` part.

File: src/modules/auto-sync-property.js [hunk 3]
```diff
@@ -31,12 +33,26 @@
 				source = scope.find(sourceSelector);
 			}
 
+			//Use property if no with property specified
+			if (withProperty.length === 0) {
```
Comment (nitriques) [inline src/modules/auto-sync-property.js]: !withProperty

File: src/modules/auto-sync-property.js [hunk 4]
```diff
@@ -31,12 +33,26 @@
...
+				if (withProperty == 'height') {
 					//Ensure to get not rounded value from jquery
-					var value = Math.floor(parseFloat(window.getComputedStyle(source[0]).height));
-					t.css({height: value});
+					value = Math.floor(parseFloat(window.getComputedStyle(source[0]).height));
+				} else if (withProperty == 'outerHeight') {
+					value = source.outerHeight();
+				} else {
+
+					return;
```
Comment (nitriques) [inline src/modules/auto-sync-property.js]: Are we missing something here ?

**Comment by nitriques** [comment]
I do not see why this can sync anything to anything ... I mostly saw the added outerHeight support...

File: src/modules/auto-sync-property.js (hunk 2 above)
Comment (PascalPiche) [inline src/modules/auto-sync-property.js]: Will cause an error on line 46

File: src/modules/auto-sync-property.js (hunk 3 above)
Comment (nitriques) [inline src/modules/auto-sync-property.js]: This is still not fixed

File: src/modules/auto-sync-property.js (hunk 1 above)
Comment (nitriques) [inline src/modules/auto-sync-property.js]: Thanks

File: src/modules/auto-sync-property.js (hunk 3 above)
Comment (nitriques) [inline src/modules/auto-sync-property.js]: ping @PascalPiche

**Pull request merged**